src/main.go

```diff
@@ -11,15 +11,15 @@ import (
 	"strings"
 	"unsafe"
 
-	"github.com/erh/gonmea/analyzer"
+	"github.com/erh/viamboat/gonmea/analyzer"
```

Comment:
could this be the issue? the path for this is actually:
`github.com/viam-labs/viamboat`
@tgritter @felixreichenbach